dvc/utils/__init__.py

```diff
@@ -398,6 +398,14 @@ def relpath(path, start=os.curdir):
     return os.path.relpath(path, start)
 
 
+def is_child_path(child, parent):
```

Comment:
I would call it `path_isin()` to be more in line with `PathInfo.isin()`.

Reply:
Renamed, thanks.